### PR TITLE
Update README: Add friendlier introduction

### PR DESCRIPTION
**Pull request by Pandapip1**
The existing introduction is somewhat scattered and not easily understandable. This reorders some paragraphs and adds additional information that may be useful.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
 
-**Before you initiate a pull request**, please read the [EIP-1](https://eips.ethereum.org/EIPS/eip-1) process document. Ideas should be thoroughly discussed on [Ethereum Research](https://ethresear.ch/t/read-this-before-posting/8) or [Ethereum Magicians](https://ethereum-magicians.org/) first.
+The goal of the EIP project is to standardize and provide high-quality documentation for Ethereum client and application protocols. This repository tracks ongoing improvements to Ethereum in the form of Ethereum Improvement Proposals (EIPs). [EIP-1](https://eips.ethereum.org/EIPS/eip-1) governs how EIPs are published.
 
-This repository tracks ongoing improvements to Ethereum. It contains:
+The [EIP status page](https://eips.ethereum.org/) tracks and lists Ethereum client and application protocols:
 
-- The [EIP status page](https://eips.ethereum.org), tracking protocols for Ethereum clients and applications
-- The [process document](https://eips.ethereum.org/EIPS/eip-1) that governs how protocols are published here
+- [Core EIPs](https://eips.ethereum.org/core) are improvements to the Ethereum execution layer, data layer, or consensus layer. These often requie a fork.
```
Comment (lightclient) [inline README.md:7]:
```suggestion
- [Core EIPs](https://eips.ethereum.org/core) are improvements to the Ethereum consensus protocol.
```

File: README.md [hunk 2]
```diff
@@ -1,19 +1,19 @@
...
+- [Networking EIPs](https://eips.ethereum.org/networking) are improvements to the Networking layer, which determines how the Ethereum peer to peer network functions.
```
Comment (lightclient) [inline README.md:8]:
```suggestion
- [Networking EIPs](https://eips.ethereum.org/networking) specify the peer-to-peer networking layer of Ethereum.
```

File: README.md [hunk 3]
```diff
@@ -1,19 +1,19 @@
...
+- [Interface EIPs](https://eips.ethereum.org/interface) are improvements to the Communication layer, which determines how conventional applications can interact with applications running on Ethereum.
```
Comment (lightclient) [inline README.md:9]:
```suggestion
- [Interface EIPs](https://eips.ethereum.org/interface) standardize interfaces to Ethereum, which determine how users and applications interact with the blockchain.
```

File: README.md [hunk 4]
```diff
@@ -1,19 +1,19 @@
...
+- [ERCs](https://eips.ethereum.org/erc) are improvements to the Application layer, which determines how applications running on Ethereum can interact with each other. These generally detail one or more application-specific interfaces.
```
Comment (lightclient) [inline README.md:10]:
```suggestion
- [ERCs](https://eips.ethereum.org/erc) specify application layer standards, which determine how applications running on Ethereum can interact with each other.
```

File: README.md [hunk 5]
```diff
@@ -1,19 +1,19 @@
...
+- [Meta EIPs](https://eips.ethereum.org/meta) are miscellaneous improvements that nonetheless require some sort of consensus. 
+- [Informational EIPs](https://eips.ethereum.org/informational) are non-standard improvements that do not require any form of consensus.
 
-For help *implementing* an EIP, please visit [Ethereum Stack Exchange](https://ethereum.stackexchange.com).
+**Before you write an EIP, ideas MUST be thoroughly discussed on [Ethereum Research](https://ethresear.ch/t/read-this-before-posting/8) or [Ethereum Magicians](https://ethereum-magicians.org/). Once consensus is reached, thouroughly read and review [EIP-1](https://eips.ethereum.org/EIPS/eip-1), which describes the EIP proess.**
```
Comment (lightclient) [inline README.md:14]:
```suggestion
**Before you write an EIP, ideas MUST be thoroughly discussed on [Ethereum Magicians](https://ethereum-magicians.org/) or [Ethereum Research](https://ethresear.ch/t/read-this-before-posting/8). Once consensus is reached, thoroughly read and review [EIP-1](https://eips.ethereum.org/EIPS/eip-1), which describes the EIP process.**
```

I think we should be pushing people to FEM as much as possible.

File: README.md [hunk 6]
```diff
@@ -1,19 +1,19 @@
...
 
-If you would like to become an EIP Editor, please check [EIP-5069](./EIPS/eip-5069.md).
```
Comment (lightclient) [inline README.md:12]: Please don't remove this.

File: README.md [hunk 7]
```diff
@@ -1,19 +1,19 @@
...
-
-## Mission
-
-The goal of the EIP project is to document standardized protocols for Ethereum clients and applications and to document them in a high-quality and implementable way.
+This repository is not for help implementing an EIP. For that, please visit the [Ethereum Stack Exchange](https://ethereum.stackexchange.com). This repository is also not for discussing EIPs. If you have a question or comment about a non-final EIP, visit its discussion page and voice your thoughts. If you have a question or comment about a final EIP, please post a new thread on [Ethereum Magicians](https://ethereum-magicians.org/).
```
Comment (lightclient) [inline README.md:16]:
```suggestion
Please note that this repository is for documenting standards and not for help implementing them. These types of inquiries should be directed to the [Ethereum Stack Exchange](https://ethereum.stackexchange.com). For specific questions and concerns regarding EIPs, it's best to comment on the relevant discussion thread of the EIP denoted by the `discussions-to` tag in the EIP's preamble.
```

**Review by lightclient — DISMISSED** [review]

**Comment by Pandapip1** [comment]
Would still like this.

CC @SamWilsn @lightclient

File: README.md [hunk 8]
```diff
@@ -1,19 +1,21 @@
 # Ethereum Improvement Proposals (EIPs)
 
-**Before you initiate a pull request**, please read the [EIP-1](https://eips.ethereum.org/EIPS/eip-1) process document. Ideas should be thoroughly discussed on [Ethereum Research](https://ethresear.ch/t/read-this-before-posting/8) or [Ethereum Magicians](https://ethereum-magicians.org/) first.
+The goal of the EIP project is to standardize and provide high-quality documentation for Ethereum client and application protocols. This repository tracks ongoing improvements to Ethereum in the form of Ethereum Improvement Proposals (EIPs). [EIP-1](https://eips.ethereum.org/EIPS/eip-1) governs how EIPs are published.
 
-This repository tracks ongoing improvements to Ethereum. It contains:
+The [EIP status page](https://eips.ethereum.org/) tracks and lists Ethereum client and application protocols:
```
Comment (SamWilsn) [inline README.md:5]:
```suggestion
The [status page](https://eips.ethereum.org/) tracks and lists EIPs, which can be divided into the following categories:
```

**Review by SamWilsn — APPROVED** [review]

**Pull request merged**